doors.lua

```diff
@@ -19,25 +19,25 @@ local scifi_nodes = {}
 scifi_nodes.doors_open_with_mesecon_only = minetest.settings:get_bool("scifi_nodes.doors_open_with_mesecon_only", false)
 
 -- Some aliases to deal with old namming policy --
-minetest.register_alias("scifi_nodes:doors_1a","scifi_nodes:Doom_door_closed")
-minetest.register_alias("scifi_nodes:doors_1b","scifi_nodes:Doom_door_closed_top")
-minetest.register_alias("scifi_nodes:doors_1c","scifi_nodes:Doom_door_opened")
-minetest.register_alias("scifi_nodes:doors_1d","scifi_nodes:Doom_door_opened_top")
-
-minetest.register_alias("scifi_nodes:doors_2a","scifi_nodes:black_door_closed")
-minetest.register_alias("scifi_nodes:doors_2b","scifi_nodes:black_door_closed_top")
-minetest.register_alias("scifi_nodes:doors_2c","scifi_nodes:black_door_opened")
-minetest.register_alias("scifi_nodes:doors_2d","scifi_nodes:black_door_opened_top")
-
-minetest.register_alias("scifi_nodes:doors_3a","scifi_nodes:white_door_closed")
-minetest.register_alias("scifi_nodes:doors_3b","scifi_nodes:white_door_closed_top")
-minetest.register_alias("scifi_nodes:doors_3c","scifi_nodes:white_door_opened")
-minetest.register_alias("scifi_nodes:doors_3d","scifi_nodes:white_door_opened_top")
-
-minetest.register_alias("scifi_nodes:doors_4a","scifi_nodes:green_door_closed")
-minetest.register_alias("scifi_nodes:doors_4b","scifi_nodes:green_door_closed_top")
-minetest.register_alias("scifi_nodes:doors_4c","scifi_nodes:green_door_opened")
-minetest.register_alias("scifi_nodes:doors_4d","scifi_nodes:green_door_opened_top")
+minetest.register_alias("scifi_nodes:doors1a","scifi_nodes:Doom_door_closed")
```

Comment:
I tested it, but the naming is actually without "s" on every door:
Here it would be: `scifi_nodes:door1a`
(sorry to be nitpicky, but i have a few worlds that depend on this awesome mod)